lib/fog/vsphere/requests/compute/list_clusters.rb

```diff
@@ -27,41 +33,41 @@ def get_raw_clusters_from_folder(folder)
           end.flatten
         end
 
-        def cluster_attributes cluster, datacenter_name
+        def cluster_attributes(cluster, datacenter_name)
           {
-            :id             => managed_obj_id(cluster),
-            :name           => cluster.name,
-            :full_path      => cluster_path(cluster, datacenter_name),
-            :num_host       => cluster.summary.numHosts,
-            :num_cpu_cores  => cluster.summary.numCpuCores,
-            :overall_status => cluster.summary.overallStatus,
-            :datacenter     => datacenter_name || parent_attribute(cluster.path, :datacenter)[1],
+            id: managed_obj_id(cluster),
```

Comment:
Oooph. Ok well I can change these back and I'll check on the opt param for 1.8.7. That explains the abundance of hash rockets. Let me know if/when you want to drop 1.8 (cause yea... reaaaaally old) and I'd be more than happy to help cleanup.